### PR TITLE
ENG-2006 Pass unbugged finished envelopes logic

### PR DESCRIPTION
**Pull request by PaulDHenson**
Added logic which allows the prev finished envelopes to be passed on without doing the game.

### CODE REVIEW

File: react/game_area.js [hunk 1]
```diff
@@ -185,7 +187,13 @@ class GameArea extends Component {
       });
       
       if (unfinishedEnvelopes.length > 0) {
-        this.createStartEnvelope(unfinishedEnvelopes);
+        if (unfinishedEnvelopes[0].prevCompleted && !unfinishedEnvelopes[0].isChanged) {
+          console.log("Inside the if statement");
```
Comment (BFisch14) [inline react/game_area.js]: Probably take out the debugging here

**Review by BFisch14 — COMMENTED** [review]
Looks good except console logs

File: react/game_area.js (hunk 1 above)
Comment (PaulDHenson) [reply react/game_area.js]: Just updated--take a look when you get a chance.

**Review by BFisch14 — APPROVED** [review]
Awesome job on this change

**Pull request merged**